src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, you're using an abstract class to define the behavior of getArea() and draw() methods. Consider whether these methods should be defined in a common interface or kept in concrete classes if they differ significantly.